tests/by-util/test_shuf.rs

```diff
@@ -624,3 +624,31 @@ fn test_shuf_multiple_input_line_count() {
         .count();
     assert_eq!(result_count, 5, "Output should have 5 items");
 }
+
+#[test]
+fn test_shuf_repeat_empty_range() {
+    new_ucmd!()
+        .arg("-ri10-3")
+        .fails()
+        .no_stdout()
+        .stderr_only("shuf: no lines to repeat\n");
+}
```

Comment:
I think this test case is incorrect. With GNU `shuf`, running `shuf -ri10-3` leads to an "invalid input range" error. I only get a "no lines to repeat" error if the difference between the first and second number is one, for example with `shuf -ri10-9`.

Reply:
Huh! You're right, that's weird. Apparently GNU shuf makes a distinction between empty and negative ranges I guess? Thanks for flagging this, I'll look into it some more.